src/_press/cal-itp-payments-data-dashboard.md

```diff
@@ -4,4 +4,5 @@ title: Cal-ITP’s free dashboard serves up ridership metrics for transit agenci
 external: https://gcn.com/data-analytics/2023/01/caltrans-serves-dashboard-metrics-local-transit-agencies/381473/
```

Comment:
```suggestion
external: https://www.route-fifty.com/digital-government/2023/01/caltrans-serves-dashboard-metrics-local-transit-agencies/381473/
```

Reply:
Confirmed this redirect is correct 👍

Reply:
@machikoyasuda I think this updated should be committed!